src/v2/view-builder/ViewFactory.js

```diff
@@ -36,7 +36,7 @@ const VIEWS_MAPPING = {
     [DEFAULT]: EnrollProfileView,
   },
   'enroll-factor': {
-    email: EnrollFactorEmailView,
+    email: RequiredFactorEmailView, // TODO EnrollFactorEmailView is unimplemented
```

Comment:
does this mean `RequiredFactorEmailView` used for both enroll and challenge?
can we just delete `EnrollFactorEmailView` and add comments why same view for both flows.

Reply:
Maybe, I thought we might make them different at some point. Somehow we messed up the API response and always returned AUTHENTICATE during email enrollment challenges so this just magically worked before.